packages/cli/src/outputer.ts

```diff
@@ -15,11 +15,20 @@
 */
 import { StepEmitter } from '@salto-io/core'
 import { EOL } from 'os'
-import { CliOutput } from './types'
+import { logger } from '@salto-io/logging'
+import { CliOutput, WriteStream } from './types'
 import { formatStepStart, formatStepCompleted, formatStepFailed } from './formatter'
 
-export const outputLine = (text: string, output: CliOutput): void => output.stdout.write(`${text}\n`)
-export const errorOutputLine = (text: string, output: CliOutput): void => output.stderr.write(`${text}\n`)
+const log = logger(module)
+
+export const outputLine = (text: string, output: CliOutput | {stdout: WriteStream}): void => {
```

Comment:
nit: since this never needs the full `CliOutput` (only needs stdout) the signature can just be
```suggestion
export const outputLine = (text: string, output: Pick<CliOutput, 'stdout'>): void => {
```

Actually, given the other comments, seems like there would be no need to change this signature because it seems like we should always be able to pass in `CliOutput`